guides/common/modules/con_updating-project-to-next-patch-version.adoc

```diff
@@ -0,0 +1,7 @@
+[id="updating-project-to-next-patch-version_{context}"]
+= Updating {Project} to the next patch version
+
+You can update your {ProjectServer} and {SmartProxyServer} to a new patch release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1, by using the {Project} maintain tool.
+The patch releases are non-disruptive to your operating environment and often fast.
+
+{Team} recommends performing updates regularly, because these releases patch security vulnerabilities and minor issues discovered after code is released.
```

Comment:
I know this was a problem before, but they don't have to be minor issues (can be major bugs, potentially even data loss bugs). They can also address known issues at the time of release.
```suggestion
{Team} recommends performing updates regularly, because these releases patch security vulnerabilities and other issues.
```